Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -278,6 +291,28 @@ public struct ConvertAction: Action, RecreatingContext {
         let temporaryFolder = try createTempFolder(
             with: htmlTemplateDirectory)
         
+        var indexHTMLData: Data?
+
+        // The `template-index.html` is a duplicate version of `index.html` with extra template
+        // tokens that allow for customizing the base-path.
+        // If a base bath is provided we will transform the template using the base path
+        // to produce a replacement index.html file.
+        // After any required transforming has been done the template file will be removed.
+        let templateURL: URL = temporaryFolder.appendingPathComponent(HTMLTemplate.templateFileName.rawValue)
+        if fileManager.fileExists(atPath: templateURL.path) {
+            // If the `transformForStaticHosting` is not set but there is a `hostingBasePath`
+            // then transform the index template
```

Comment:
I'm not sure I understand this behavior. Why are we modifying the `index.html` if `transformForStaticHosting` is not set?

Reply:
We still transform the template if `hostingBasePath` has been set to allow the non static version of the archive to be hosted somewhere other than root.

Reply:
This change was based on some feedback we got from Konrad in the forums: https://forums.swift.org/t/support-hosting-docc-archives-in-static-hosting-environments/53572/7.

The `hosting-base-path` parameter isn't specific to the `transform-for-static-hosting` work we're doing so it makes sense to support it here as well.